doc/manual/manual.tex

```diff
@@ -2625,118 +2625,56 @@ \subsubsection{Running \aspect{} models}
-prerequisites:
 \begin{enumerate}
-\item \textit{Trilinos:} Trilinos can be downloaded from
-  \href{http://trilinos.org/download/}{http://trilinos.org/download/}. At
-  the current time we recommend Trilinos Version 11.4.x.%
-  \footnote{Other versions of Trilinos like 10.6.x
-  and 10.8.x have bugs that make these versions unusable for our purpose. The
-  \dealii{} ReadMe file provides a list of versions that are known to work
-  without bugs with \dealii{}.} For installation instructions see
-  \href{https://www.dealii.org/developer/external-libs/trilinos.html}{the deal.II README file on installing Trilinos}. Note that you have
-  to configure with MPI by using
-\begin{verbatim}
- -DTPL_ENABLE_MPI:BOOL=ON
-\end{verbatim}
-  in the call to cmake. After that, run {\tt{make install}}.
-
-\item \textit{\pfrst{}:} Download and install \pfrst{} as described in the
-  \href{https://www.dealii.org/developer/external-libs/p4est.html}{deal.II
-    p4est installation instructions}. This is done using the
-  {\tt{p4est-setup.sh}}; do not use the \pfrst{} stand-alone installation
-  instructions.
-
-\item \textit{\dealii{}:}
-  The current version of \aspect{} requires \dealii{} version 8.4.0 or later.
-  This version can be downloaded and installed from
-  \url{https://www.dealii.org/download.html}.
-
-\item \textit{Configuring and compiling \dealii:} Now it is time to configure
-  \dealii.
-  To this end, follow the \dealii{}
-  \href{https://www.dealii.org/developer/readme.html}{installation
-    instructions}. Note that \dealii{} recently made the switch to cmake,
-    so the configuration changed. Make sure you enable MPI.
-    A typical command line would look like this:
-\begin{verbatim}
-mkdir build
-cd build
-cmake -DDEAL_II_WITH_MPI=ON \
-      -DCMAKE_INSTALL_PREFIX=/u/username/deal-installed/ \
-      -DTRILINOS_DIR=/u/username/trilinos-11.4.1/ \
-      -DP4EST_DIR=/u/username/p4est-0.3.4.1/ \
-      ../deal.II
-\end{verbatim}
-  if the Trilinos and \pfrst{} packages have been installed in the
-  subdirectory \texttt{/u/username/}.
-  Make sure the configuration succeeds and detects the MPI compilers
-  correctly. For more information see the documentation of \dealii.
+\item \textit{Obtaining candi:} Download \texttt{candi} by running 
+    \begin{verbatim}
+    git clone https://github.com/dealii/candi
+    \end{verbatim} 
+    in a directory of your choice. 
 
-  Now you are ready to compile \dealii{} by running {\tt{make install}}. If you
-  have multiple processor cores, feel free to do {\tt{make install -jN}} where
-  \texttt{N} is the number of processors in your machine to accelerate the
-  process.
+\item \textit{Installing \dealii{} and its dependencies:} Execute \texttt {candi} by running
+    \begin{verbatim}
+    cd candi
+    ./candi.sh -p INSTALL_PATH
+    \end{verbatim} 
+    (here we assume you replace INSTALL\_PATH by the path were
+    you want to install all dependencies and \dealii{}). 
```

Comment:
Maybe add "A typical directory for this would be \texttt{$HOME/bin} or a similar place somewhere under your home directory."